docs/source/getting-started/content/proc_troubleshooting-bundle-version-mismatch.adoc

```diff
@@ -0,0 +1,31 @@
+[id="troubleshooting-bundle-version-mismatch_{context}"]
+= Troubleshooting bundle version mismatch
+
+As {prod} releases are set up and started, instances are created with bundle information and instance data.  When a new {prod} release is setup, the bundle information and instance data is not updated due to the customization in the previous instance data.  This will lead to errors on [command]`{bin} start` command. The error will be similar to this:
+
+----
+$ {bin} start
+INFO Checking if running as non-root
+INFO Checking if oc binary is cached
+INFO Checking if HyperKit is installed
+INFO Checking if crc-driver-hyperkit is installed
+INFO Checking file permissions for /etc/resolver/testing
+INFO Checking file permissions for /etc/hosts
+FATA Bundle 'crc_hyperkit_4.2.8.crcbundle' was requested, but the existing VM is using
+'crc_hyperkit_4.2.2.crcbundle'
+----
+
+.Procedure
+
+To resolve the error, issue the [command]`{bin} delete` command before attempting to start the instances:
+
+[subs="+quotes,attributes"]
+----
+$ {bin} delete
+----
+
+[NOTE]
+====
+The [command]`{bin} delete` command will result in the loss of data stored in the {prod} virtual machine. Steps should be taken to save any instance information before issuing the command. 
```

Comment:
Likewise, each sentence on its own line, please. :)

Reply:
Moved sentences to their own line.